stock_production_lot_multi_company/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 Ainara Galdona - AvanzOSC
+# Copyright 2017 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
```

Comment:
Can you update this with the new template? https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py 

Add the website key for example,